src/formatDate.js

```diff
@@ -8,7 +8,57 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromValue = fromFormat[fromFormat.length - 1];
+
+  const toValue = toFormat[toFormat.length - 1];
```

Comment:
Similar to `fromValue`, `toValue` assumes the last element of `toFormat` is a separator. Ensure that this assumption holds or add validation to handle different cases.